package.json

```diff
@@ -33,7 +33,7 @@
   },
   "dependencies": {
     "@webex/common": "^1.105.0",
-    "@webex/component-adapter-interfaces": "^1.11.0"
+    "@webex/component-adapter-interfaces": "latest"
```

Comment:
Can you coordinate with @cricku? He's also changing this dependency in his PR https://github.com/webex/sdk-component-adapter/pull/102